src/inverseRobot.js

```diff
@@ -4,10 +4,20 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
```

Comment:
your variables should have more descriptive names, so you should avoid naming them like `result` or `object`.
What about `inversedRobot`?